### PR TITLE
simulate effect of amplification on mean and variance

### PR DESCRIPTION
**Pull request by frederic-mahe**
A solution using `purrr`, starting from simple functions and combining them to build a pipeline simulating the sampling, amplification, and sequencing of 4 species, in 79 samples, and with 2 methods (metabarcoding, metagenomics).

I've added an extra-step (`many_runs`) that generates the results 10 times.

### CODE REVIEW

File: effect_of_amplification_on_mean_and_variance.R [hunk 1]
```diff
@@ -0,0 +1,79 @@
+library(vegan)
+library(tidyverse)
+
```
Comment (GhMathis) [inline effect_of_amplification_on_mean_and_variance.R:4]: add the date

**Review by GhMathis — CHANGES_REQUESTED** [review]
Test request change

File: effect_of_amplification_on_mean_and_variance.R (hunk 1 above)
Comment (frederic-mahe) [reply effect_of_amplification_on_mean_and_variance.R:4]: done

**Review by GhMathis — APPROVED** [review]
All good, merging to main

**Pull request merged**